internal/driver/glfw/window_test.go

```diff
@@ -821,6 +821,42 @@ func TestWindow_Clipboard(t *testing.T) {
 	cb.SetContent(cliboardContent)
 }
 
+func TestWindow_WillClose(t *testing.T) {
+	closed := make(chan bool)
```

Comment:
I think we can simplify this test just using a boolean variable. Indeed it will be set in a closure passed by `SetOnClosed()`.

Reply:
i did't manage to get stable tests using simple variable, running single test everything is OK but running the full set was not stable

Reply:
Probably that's because of goroutines involved. Can you try use atomics instead?

Reply:
Yes, I think hte goroutines are adding code complexity - this is what my alternative suggestion was attempting to avoid.